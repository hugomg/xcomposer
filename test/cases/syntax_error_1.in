0x0041 A A