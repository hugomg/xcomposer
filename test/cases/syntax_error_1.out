xcomposer encountered an error:

Syntax error in Line 1: bad rule
